src/test/java/scenarios/CustomersTest.java

```diff
@@ -0,0 +1,340 @@
+package scenarios;
+
+import org.automation.base.BaseTest;
+import org.automation.data.KadeUserAccount;
+import org.automation.session.KadeSession;
+import org.automation.utilities.Assertions;
+import org.automation.utilities.WebdriverWaits;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class CustomersTest extends BaseTest {
+
+    @Test(description = "Verify that Store's customers page opens up displaying all the options")
+    public void c_01storeCustomersPage() {
+        KadeSession session = KadeSession.login(KadeUserAccount.Default);
+        session.getSidePannel().expandManageBusinessAccordionBttn().click();
+        session.getSidePannel().getCustomersTab().click();
+        session.getCustomersPage().storeSelection();
+        session.getCustomersPage().continuebtn().click();
+
+        Assertions.assertTrue(session.getCustomersPage().nameAddress().isDisplayed());
+        Assertions.assertTrue(session.getCustomersPage().findAddCustomer().isDisplayed());
+        Assertions.assertTrue(session.getCustomersPage().filter().isDisplayed());
+    }
+
+    @Test(description = "Adding a new customer with Phone number")
+    public void c_02addCustomerWithPhoneNumber() {
+
+        KadeSession session = KadeSession.login(KadeUserAccount.Default);
+        session.getSidePannel().expandManageBusinessAccordionBttn().click();
+        session.getSidePannel().getCustomersTab().click();
+        session.getCustomersPage().storeSelection();
+        session.getCustomersPage().continuebtn().click();
+
+        session.getCustomersPage().findAddCustomer().click();
+        WebDriverWait wait = new WebDriverWait(getDriver(), Duration.ofSeconds(10));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//span[@class='me-1']")));
+        System.out.println(getDriver().findElement(By.xpath("//span[@class='me-1']")).getText());
+        session.getCustomersPage().setPhoneNumber("9011017524");
+        session.getCustomersPage().goPhoneNumber().click();
+//        WebdriverWaits.sleep(2000);
+//        session.getCustomersPage().setCustomerName("Rishabh");
+//        session.getCustomersPage().doneBtn().click();
+
+        session.getCustomersPage().findAddCustomer().click();
+        WebDriverWait waitt = new WebDriverWait(getDriver(), Duration.ofSeconds(10));
+        waitt.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//span[@class='me-1']")));
+        session.getCustomersPage().getPhoneField().setText("90110");
+        session.getCustomersPage().goPhoneNumber().click();
+        Assertions.assertTrue(session.getCustomersPage().getPhoneValidation().isDisplayed());
```

Comment:
Boundary Values: Ensure edge cases such as invalid phone numbers and empty inputs are tested.
Improvement Suggestion: Parameterize test cases using @DataProvider to handle multiple inputs efficiently.

Reply:
Thank you for the suggestion! I've updated the code to use the @DataProvider annotation for parameterizing the test cases. This should now handle multiple inputs more efficiently.